core/src/main/java/com/yahoo/oak/EntryOrderedSet.java

```diff
@@ -294,10 +295,13 @@ boolean copyEntry(ValueBuffer tempValue, EntryOrderedSet<K, V> srcEntryOrderedSe
     boolean isEntrySetValidAfterRebalance() {
```

Comment:
this resulted in rare fail in the OrderedChunk assert, is the new conditions ok?